src/stores/main/actions.ts

```diff
@@ -417,7 +417,9 @@ export function fetchPermissions(
   return async (dispatch, getState) => {
     const orgs = Object.values(selectOrganizations()(getState()));
 
-    const namespaces = orgs.map((o) => o.namespace ?? '');
+    const namespaces = orgs.map(
+      (o) => o.namespace ?? getNamespaceFromOrgName(o.id)
```

Comment:
If this is empty, doesn't it mean it isn't reconciled and the ns may be not even created?

Reply:
Sure, that might be the case, and the request would fail. But with the new logic, it doesn't break anything